wire-ios-notification-engine/Sources/CoreCrypto/NotificationSession+CoreCrypto.swift

```diff
@@ -58,4 +62,16 @@ extension NotificationSession {
         }
     }
 
+    private var shouldSetupCryptoStack: Bool {
+        return shouldSetupProteusService || shouldSetupMLSController
+    }
+
+    private var shouldSetupProteusService: Bool {
+        return DeveloperFlag.proteusViaCoreCrypto.isOn
+    }
+
+    private var shouldSetupMLSController: Bool {
+        return DeveloperFlag.enableMLSSupport.isOn && (BackendInfo.apiVersion ?? .v0) >= .v2
```

Comment:
what if we want to try mls, we cannot just toggle the flag and select the preferred Api Version, it can only work with the backend enabling it, right?

Reply:
Yeah I think there's a backend side to this as well, but we'll need to address that in the future. For now this is just to make sure that we don't run mls code unless we explicitly want to.